js/language_server/src/tests/runTests.ts

```diff
@@ -0,0 +1,32 @@
+import * as path from "path";
+import find = require("find-process");
+
+import { runTests } from "@vscode/test-electron";
+
+async function go() {
```

Comment:
there isn't a cleaner way to get xvfb's DISPLAY?

Reply:
I don't think so, this was copy pasted from the official vscode language server repo 🤷 so